packages/components/src/FormField/FormField.module.css

```
@@ -117,7 +117,7 @@
 .disabled {
   --field--placeholder-color: var(--color-disabled);
   --field--value-color: var(--color-disabled);
-  --field--background-color: var(--color-surface--background);
+  --field--background-color: var(--color-interactive--background);
   --field--border-color: var(--color-disabled--secondary);
```

```suggestion
  --field--background-color: var(--color-disabled--secondary);
  --field--border-color: var(--color-border);
```

This has been a long-standing UI bug, our disabled inputs should use the disabled color for their background!